study.md

```diff
@@ -31,7 +38,11 @@ In your own words explain "this".  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+"this" is a property of a function, that holds the value of an object. Whenever
+you use this in a function it is going to get the value of a variable previously
+declared.
```

Comment:
Close here just wanna make sure you get this straight. When a method is called on an object, inside the method `this` will refer to the object you called the method on.